sentry-sidekiq/lib/sentry/sidekiq/sentry_context_middleware.rb

```diff
@@ -2,20 +2,29 @@
 
 module Sentry
   module Sidekiq
-    class SentryContextMiddleware
+    class SentryContextServerMiddleware
+      def initialize(testing_only_callback: nil)
+        @testing_only_callback = testing_only_callback
```

Comment:
The only way I can imagine was to pass a callback. Is there a better way?

Reply:
I've left a comment for a better approach below. I think this hack is unnecessary.